bodhi/server/scripts/dequeue_stable.py

```diff
@@ -0,0 +1,40 @@
+# This program is free software; you can redistribute it and/or
```

Comment:
Let's add a Copyright claim here. Only, I'm not quite sure who to put as the Copyright holder. I started a thread on outreachy-list to ask. If you are on that list, perhaps you can use their feedback to know what to do here. Anyways, once we know who or what organization to give the Copyright to, let's do something like this here:

```
# -*- coding: utf-8 -*-
# Copyright © 2017 <copyright holder here>
#
# This file is part of Bodhi.
#
```

and then the rest of the header like you have it here.